keyboard/android/src/main/java/com/capacitorjs/plugins/keyboard/KeyboardPlugin.java

```diff
@@ -75,9 +75,11 @@ void onKeyboardEvent(String event, int size) {
         switch (event) {
             case Keyboard.EVENT_KB_WILL_SHOW:
             case Keyboard.EVENT_KB_DID_SHOW:
-                String data = "{ 'keyboardHeight': " + size + " }";
+                // Convert Handler().postDelayed() 350ms to 0.35s
+                String data = "{ 'keyboardHeight': " + size + ", 'keyboardAnimationDuration': " + 0.35 + " }";
```

Comment:
Thanks so much for contributing this. Is 350 a constant on Android?

Reply:
```
@PluginMethod
    public void show(final PluginCall call) {
        execute(
            () ->
                new Handler()
                    .postDelayed(
                        () -> {
                            implementation.show();
                            call.resolve();
                        },
                        350
                    )
        );
    }
```
350 is method param `long delayMillis`

Reply:
I'm sorry, I meant: on Android, is 350 milliseconds always the time it takes for the keyboard animation to complete. I'm wondering why that number is hard-coded.

Thank you.

Reply:
I may be speaking out of ignorance here, but is this value only useful on iOS?  Perhaps the android part of this should be removed, and the docs should specify this as an iOS only value for the plugin?